vulnerabilities/severity_systems.py

```diff
@@ -161,3 +161,121 @@ def compute(self, scoring_elements: str) -> str:
         APACHE_TOMCAT,
     )
 }
+
+
+def convert_cvssv2_vector_to_cvssv3_vector(cvssv2_vector):
```

Comment:
Can you extract this function in a seperate file?

Reply:
Done